mltrace/entities/components.py

```diff
@@ -10,13 +10,13 @@
 
 class PreprocessingComponent(Component):
```

Comment:
Let's delete this file as the `Preprocessing` component is never actually used / we haven't started working on the library yet

Reply:
removed